### PR TITLE
Major python code change

### PR DESCRIPTION
**Pull request by Roosted7**
Refractoring splash code

### CODE REVIEW

**Comment by Roosted7** [comment]
Do _NOT_ merge yet :)

File: esp32/modules/splash.py [hunk 1]
```diff
@@ -192,360 +74,221 @@ def draw_helper_battery(percent,cstate):
             else:
                 ugfx.string(5,5,"empty","Roboto_Regular12",header_bg)
     else:
-        ugfx.string(2,5,"no batt","Roboto_Regular12",header_bg)
-    
+        ugfx.string(2,5,"no batt","Roat_status = boto_Regular12",header_bg)
```
Comment (raboof) [inline esp32/modules/splash.py]: A stray `at_status = ` appeared :)

**Comment by Roosted7** [comment]
So, this now is flashable and kinda usable...
Will sort out the bugs later to make this mergable

File: esp32/modules/appglue.py [hunk 2]
```diff
@@ -2,11 +2,12 @@
 
 def start_app(app):
     ugfx.clear(ugfx.WHITE)
-    ugfx.string(0, 0, "Please wait...", "PermanentMarker22", ugfx.BLACK)
+    ugfx.string(0, 0, "Rebooting..", "PermanentMarker22", ugfx.BLACK)
     if (app==""):
         ugfx.string(0,  25, "Returning to homescreen...","Roboto_Regular12",ugfx.BLACK)
     else:
         ugfx.string(0,  25, "Starting "+app+"...","Roboto_Regular12",ugfx.BLACK)
+    ugfx.set_lut(ugfx.LUT_FASTEST)
```
Comment (raboof) [inline esp32/modules/appglue.py]: You don't have to `set_lut` separately, you can pass `ugfx.LUT_FASTEST` to `ugfx.flush()`

**Comment by renzenicolai** [comment]
Hmm, I misunderstood bpp then. We do need a way to set the maximum time bpp will be active though.

File: esp32/modules/appglue.py (hunk 2 above)
Comment (Roosted7) [reply esp32/modules/appglue.py]: Will do!

**Comment by Kartoffel** [comment]
Please make sure the badge does not go to sleep when plugged in through USB, this makes the REPL a bit difficult to use

**Comment by Kartoffel** [comment]
The splash may take a bit longer refreshing after a reset, [it can be a bit ugly when exiting an app](http://i.imgur.com/c4zQ8u5.png).

Have you thought about doing a nice and slow refresh before going to auto-sleep?

**Comment by Kartoffel** [comment]
Pressing "B" to launch the OTA update does not work.

Change label "[B: OTA]" --> "[B: Update]"

**Comment by Kartoffel** [comment]
Tiny bug, it appears that my battery is a bit full. [LOL](http://i.imgur.com/naZWqAu.png).

(At 100% or 4.2V)

Unplugging charger reports a voltage of 4.31V

**Pull request merged**